modules/installation/pages/ports.adoc

```diff
@@ -146,4 +146,7 @@ In most cases, you will not need to adjust these ports.
 | 22          | Inbound   | SSH      | Required for ssh-push and ssh-push-tunnel contact methods.
 | 80          | Outbound  |          | Used to reach the server or proxy.
 | 5222        | Outbound  | TCP      | Required to push OSAD actions to the server or proxy.
+| 9100        | Outbound  | TCP      | Required for Prometheus node exporter
+| 9117        | Outbound  | TCP      | Required for Prometheus Apache exporter
+| 9187        | Outbound  | TCP      | Required for Prometheus PostgreSQL
```

Comment:
Nit: Points missing at the ends of the sentences.

Reply:
To be fair, we should probably remove the periods from all of them, because they're not full sentences, but then I think @keichwa would probably come after me :wink: